lib/sarif/npm_audit_sarif.rb

```diff
@@ -0,0 +1,49 @@
+module Sarif
+  class NPMAuditSarif < BaseSarif
+    NPM_URI = 'https://docs.npmjs.com/cli/v7/commands/npm-audit'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = NPM_URI
+      @logs = @scan_report.to_h[:info][:stdout][:advisories].values
```

Comment:
What is scanned failed with parse error?  Then `scan_report` will not contain `:advisories`.

Can you try manually add weird characters in package lock json so that there will be a parse error?  Set salus.yaml to write to a json and see how the hash key/vals look like.

Reply:
I've handled the parse errors generated. They're logged in the invocation object